.github/workflows/flannel.yml

```diff
@@ -0,0 +1,17 @@
+name: flannel-images
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  build:
+    runs-on: windows-2019
+    steps:
+    - uses: actions/checkout@v2
+    - uses: azure/docker-login@v1
+      with:
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_SECRET }}
+    - name: Build
+      run: go run kubeadm/hack/publish-flannel.go kubeadm/flannel
```

Comment:
Is there a reason we a binary to build and publish the image over just running docker build/push commands?

Reply:
As I understand it required to build docker images with tags similar to flannel versions.
Anyway this file is modified version of `.github/workflows/kube-proxy.yml`

Reply:
I believe it was added to automatically look up Kubernetes releases and publish new kube-proxy. In this case flannel doesn't change often and it's more likely a fix in our files that would cause a new image (like in #106) so I think we don't need this extra look up

Reply:
Do I need to do any additional changes except trigger settings?

Reply:
If we change the code `go run kubeadm/hack/publish-flannel.go kubeadm/flannel` to `docker build/docker push` then we need to update the code after each new flannel version.
I changed workflow trigger as you said. So flannel images will be updated only after future commit in master.

Reply:
This will build and push for each version greater than 0.12? from that perspective I think it would be useful to build each of the versions with new scripts. So I guess I am convinced. :-)

Reply:
Yep. This will build tags for each flannel version greater than 0.12 and replace previous with same version.
This is the same situation as for kube-proxy. If we change Dockerfile for it next day all tags will be replaced.